src/elements/content-sidebar/activity-feed/task-new/AvatarGroupAvatar.js

```diff
@@ -40,13 +40,19 @@ const AvatarGroupAvatar = React.memo<Props>(({ user, status, getAvatarUrl, class
         data-testid="avatar-group-avatar-container"
         {...rest}
     >
-        <Avatar className="bcs-AvatarGroupAvatar-avatar" user={user} getAvatarUrl={getAvatarUrl} />
-        <StatusIcon
-            status={status}
-            className={`bcs-AvatarGroupAvatar-statusIcon ${camelCase(status)}`}
-            height={12}
-            width={12}
-            title={<FormattedMessage {...messages.taskAssignmentCompleted} />}
+        <Avatar
+            badgeIcon={
+                <StatusIcon
+                    className={`${camelCase(status)}`}
+                    height={12}
```

Comment:
Can we standardize the size of the activity type badges? Can we use 20/20 for this one?